mlibrary/my_account
# Design System `m-callout`

# Overview
Since the launch of [Account](https://account.lib.umich.edu/), the Design System has created an [`m-callout` component](https://github.com/mlibrary/design-system/tree/main/src/components/callout) that mimics what was originally created on the Account site, and the [Library site](https://www.lib.umich.edu/locations-and-hours). All instances of the original callout component created for Account has been replaced with the Design System component.

## Testing
* Run tests to make sure they pass (`docker-compose run web bundle exec rspec`)
* Go through the site to make sure nothing is broken
* Navigate to [Account Overview](http://localhost:4567/) to check these areas to see the new callout:
  * `Design & Development Options`
  * [Current Checkouts > U-M Library](http://localhost:4567/current-checkouts/u-m-library):
    * "Items checked out prior to July 15, 2021..."
    * `Renew all eligible` confirmation
